packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -861,47 +927,100 @@ const runGenerator = (context, {outputDir, templateVersion, verbose}) => {
                 })
         }
 
-        // Process selected Application Extensions
-        processAppExtensions(selectedAppExtensions, extractAppExtensions, appExtensionsDir)
+        // Check project type and handle appropriately
+        if (answers.project.type === 'appExtensionProject') {
+            const devOutputDir = p.join(outputDir, APP_EXTENSIBILITY_TYPESCRIPT_MINIMAL_DEV_DIR)
+
+            // Update the root package.json to add a start script
+            updatePackageJson(p.resolve(outputDir, 'package.json'), {
+                scripts: {
+                    start: 'npm --prefix ./dev start'
+                }
+            })
+
+            // Recursively call runGenerator for the 'typescript-minimal' preset
+            const extensionContext = {
+                ...context,
+                preset: {
+                    id: 'typescript-minimal',
+                    templateSource: {type: TEMPLATE_SOURCE_BUNDLE, id: 'typescript-minimal'},
+                    private: true
+                },
+                answers: {project: {type: 'PWAKitProject', name: 'typescript-dev'}}
+            }
+
+            await runGenerator(extensionContext, {
+                outputDir: devOutputDir,
+                templateVersion,
+                verbose,
+                isRecursiveCall: true
```

Comment:
`isRecursiveCall` is very specific to the use-case of wanting to generate one project after another. I think it'll be better to get more generic as here and call this variable something like "installDependencies: false" or something like that so it's a little more self documenting. Unless there is some future plans that make `isRecursiveCall` a better choice. Then please let me know.